src/message.js

```diff
@@ -295,9 +312,10 @@ Message.prototype.encrypt = function(keys, passwords, sessionKey) {
  * @param  {String} symAlgo            session key algorithm
  * @param  {Array<Key>} publicKeys     (optional) public key(s) for message encryption
  * @param  {Array<String>} passwords   (optional) for message encryption
+ * @param  {Boolean} useWildcard       (optional) if a key ID wildcard of 0 should be used instead of the public key IDs
```

Comment:
confusing syntax in description, maybe 'use a key ID of 0 instead of the public key IDs'